### PR TITLE
Milestone messages

### LINKED ISSUE #197: Bot should congratulate users every 50 wins
Do like a MOD(50) or something and congratulate users when they hit a milestone.

We could expand this out to different types of milestones and differing messages/etc!

### PR DESCRIPTION
**Pull request by michael-robbins**
This adds in custom milestone messages

Fixes #197

### CODE REVIEW

File: emojirades/commands/gamestate_commands/correct_guess_command.py [hunk 1]
```diff
@@ -83,10 +83,31 @@ def execute(self):
                 "Holy bejesus Batman :bat::man:, they guessed it in one go! :clap:",
             )
 
+        # Build the score message
+        if score == 1000:
+            prefix = (
+                f"Ok <@{state['winner']}> just give up already, you've won the game"
+            )
```
Comment (phelios) [inline emojirades/commands/gamestate_commands/correct_guess_command.py:90]: what's with the brackets here?

File: emojirades/commands/gamestate_commands/inferred_correct_guess_command.py [hunk 2]
```diff
@@ -80,10 +83,32 @@ def execute(self):
...
+        elif score == 300:
+            prefix = f"Third century's the charm they say <@{state['winner']}>, congrats :sunglasses:"
+        elif score == 200:
+            prefix = f"Not going to lie <@{state['winner']}> this is pretty impressive"
+        elif score == 100:
+            prefix = f"Triple digits <@{state['winner']}>! Not everyone makes it this far! :tada:"
+        elif score % 50 == 0:
+            prefix = f"Another day, another 50 point milestone for <@{state['winner']}> :chart_with_upwards_trend:"
+        else:
+            prefix = f"Congrats <@{state['winner']}>"
```
Comment (phelios) [inline emojirades/commands/gamestate_commands/inferred_correct_guess_command.py:105]: come on! this is duplicated! extract into a function or something? and add unit tests! :P

Comment (phelios) [reply emojirades/commands/gamestate_commands/inferred_correct_guess_command.py:105]: or maybe this whole inferred class could extend the other one.

File: emojirades/commands/gamestate_commands/correct_guess_command.py (hunk 1 above)
Comment (michael-robbins) [reply emojirades/commands/gamestate_commands/correct_guess_command.py:90]: Black decided to do this to put the string on a new line

File: emojirades/commands/gamestate_commands/inferred_correct_guess_command.py (hunk 2 above)
Comment (michael-robbins) [reply emojirades/commands/gamestate_commands/inferred_correct_guess_command.py:105]: Yeah..... was kinda cbf on that.... raise #204 to track this

**Pull request merged**